types/office-js-preview/index.d.ts

```diff
@@ -119630,575 +119598,523 @@ declare namespace Visio {
-        /**
-         *
-         * If true then connectors are added in sequence according to the uniqueId column.
-         *
-         * [Api set:  1.1]
-         */
-        connectSequentially: boolean;
-        /**
-         *
-         * Delimiter for TargetColumn. It should not have more then one character.
-         *
-         * [Api set:  1.1]
-         */
-        delimiter: string;
-        /**
-         *
-         * Direction of connector to use in diagram.
-         *
-         * [Api set:  1.1]
-         */
-        direction: Visio.ConnectorDirection | "FromTarget" | "ToTarget";
-        /**
-         *
-         * Column for ConnectorLabel (shape text of connector).
-         *
-         * [Api set:  1.1]
-         */
-        labelColumnName: string;
-        /**
-         *
-         * Column for proving connected target shape.
-         *
-         * [Api set:  1.1]
-         */
-        targetColumnName: string;
-    }
-    /**
-     *
      * Represents the Horizontal Alignment of the Overlay relative to the shape.
      *
+     * @remarks
      * [Api set:  1.1]
      */
     enum OverlayHorizontalAlignment {
         /**
-         *
          * left
-         *
+         * @remarks
+         * [Api set:  1.1]
          */
         left = "Left",
         /**
-         *
          * center
-         *
+         * @remarks
+         * [Api set:  1.1]
          */
         center = "Center",
         /**
-         *
          * right
-         *
+         * @remarks
+         * [Api set:  1.1]
          */
         right = "Right",
     }
     /**
-     *
      * Represents the Vertical Alignment of the Overlay relative to the shape.
      *
+     * @remarks
      * [Api set:  1.1]
      */
     enum OverlayVerticalAlignment {
         /**
-         *
          * top
-         *
+         * @remarks
+         * [Api set:  1.1]
          */
         top = "Top",
         /**
-         *
          * middle
-         *
+         * @remarks
+         * [Api set:  1.1]
          */
         middle = "Middle",
         /**
-         *
          * bottom
-         *
+         * @remarks
+         * [Api set:  1.1]
          */
         bottom = "Bottom",
     }
     /**
-     *
      * Represents the type of the overlay.
      *
+     * @remarks
      * [Api set:  1.1]
      */
     enum OverlayType {
         /**
-         *
          * text
-         *
+         * @remarks
+         * [Api set:  1.1]
          */
         text = "Text",
         /**
-         *
          * image
-         *
+         * @remarks
+         * [Api set:  1.1]
          */
         image = "Image",
         /**
-         *
          * html
-         *
+         * @remarks
+         * [Api set:  1.1]
          */
         html = "Html",
     }
     /**
-     *
      * Toolbar IDs of the app
      *
+     * @remarks
      * [Api set:  1.1]
      */
     enum ToolBarType {
         /**
-         *
          * CommandBar
```

Comment:
Should we make these descriptions not repeats of the respective values? (Here and other enums) e.g.:

```suggestion
         * The command bar toolbar type.
```

Reply:
I'd be more inclined to remove the superfluous descriptions.